ui/app/styles/utils/z-indices.scss

```diff
@@ -5,3 +5,7 @@ $z-gutter-backdrop: 219;
 $z-subnav: 200;
 $z-base: 100;
 $z-icon-decorators: 50;
+
+$z-zero: 0;
+$z-below: -1;
+$z-below-below: -2;
```

Comment:
I needed some z-index manipulation so I could have `::before` cover up `::after` if the text wasn’t actually overflowing. Another instance where I’m open to suggestions for better names.